src/Components/LoginPage/LoginPageComponents/ShowPasswordButton.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+export const ShowPasswordButton = (props) => {
+
+    let eyeButtonClassName = 'password-block__eye-button'
+
+    if (props.inputId === "password")
+        if (props.state.signFormShowPassword) {
+            eyeButtonClassName += " password-block__eye-button_show"
+        } else {
+            eyeButtonClassName += " password-block__eye-button_hide"
+        }
```

Comment:
```
const isPasswordField = props.inputId === "password" || props.inputId === "confirmPassword";
const eyeButtonClassName = 'password-block__eye-button';
const eyeButtonVisibleClassName = `${eyeButtonClassName}{isPasswordField ? '_hide' : '_show'}`

const fullClassName = ....
```

Reply:
Code above is just a variant but looks like your Show button should have no functionality, only be responsible for displaying